### PR TITLE
SAM4N support

### LINKED ISSUE #7: Add support for SAM4N
SVD files can be found https://www.keil.com/dd2/pack/ under Microchip SAM4 Series Device Support and Examples.

- [ ] Make sure svd2rust generates correctly
- [ ] Add each target to top-level Cargo.toml
- [ ] Ensure doc.rs link is valid for each target mcu
- [ ] Publish to crates.io
- [ ] Ensure README.md and Cargo.toml have the correct target mcu

### PR DESCRIPTION
**Pull request by john-terrell**
Fixes #7

### CODE REVIEW

File: pac/atsam4n16b/device.x [hunk 1]
```diff
@@ -0,0 +1,21 @@
+PROVIDE(PMC = DefaultHandler);
```
Comment (haata) [inline pac/atsam4n16b/device.x:1]: Has same problems as #14

**Review by haata — CHANGES_REQUESTED** [review]
Looks good otherwise

File: pac/atsam4n16b/device.x (hunk 1 above)
Comment (john-terrell) [reply pac/atsam4n16b/device.x:6]: I've modified the .xsl's to include a base atsam4.xsl with the common interrupt definitions.

**Review by haata — APPROVED** [review]
:+1:

**Pull request merged**